lighthouse-core/audits/multi-check-audit.js

```diff
@@ -53,6 +53,7 @@ class MultiCheckAudit extends Audit {
     if (result.failures.length > 0) {
       return {
         score: 0,
+        // TODO(exterkamp): make this i18n-able.
```

Comment:
Afaict we don't ever really use "failures" because it is just an auto combination of a list of strings into an explanation.  I think we just use explanations manually in my experience.  We might want to just migrate away from failures totally since this is all they do.

I added this to keep the PR scoped, but I think there is opportunity here for better i18n.

Reply:
are we tracking this in a mega i18n issue somewhere? if we intend to collect TODOs in the code, I wonder if it might be better to make them all `TODO(i18n)` or something easily findable when we need to make our next i18n impacting decisions

Reply:
ping :)

Reply:
+1 we need a way to track this. I was also told that #7238 is super important for tracking this stuff, so let's actually track some of this stuff in there :P